cost-analyzer/values.yaml

```diff
@@ -2258,6 +2258,11 @@ kubecostAggregator:
   # `deployMethod: "statefulset"`
   replicas: 1
 
+  # Collections queries are cached in memory. Turning it off is not recommended
```

Comment:
Can you explain what collections are very simply?

Reply:
@kwombach12 care to weigh in? Haha.